src/main/resources/application.properties

```diff
@@ -16,3 +16,11 @@ spring.flyway.locations=classpath:db/migration
 spring.flyway.baseline-on-migrate=true
 spring.flyway.validate-on-migrate=true
 spring.flyway.clean-on-validation-error=true
+#mustache
+spring.mustache.expose-request-attributes=true
+#validation
+Required = This field is required.
+Size.userForm.username = Username must be between 8 and 32 characters.
+Duplicate.userForm.username = Such username already exists.
+Size.userForm.password = Password must be over 8 characters.
+Different.userForm.password = Password don't match.
```

Comment:
``
Required = This field is required.
Size.userForm.username = Username must be between 8 and 32 characters.
Duplicate.userForm.username = Such username already exists.
Size.userForm.password = Password must be over 8 characters.
Different.userForm.password = Password don't match. 
``
why is it here?